app/src/Manager/AdvisorManager.php

```diff
@@ -24,22 +27,52 @@ private function post($body): Response
     }
 
     /**
-     * Return the team of the player stored in the sessoin
+     * Return the best advice for the player
      *
      * @param  Game $game
      *
      * @return int
      */
-    public function getAdvice(Game $game)
+    public function getAdvice(Game $game): array
     {
         $body = [
             'Grid' => $game->getBoard(),
             'Player' => $game->getCurrentPlayer(),
         ];
         $response = $this->post(json_encode($body));
-        $body = $response->body;
 
-        $advice = [
+         return $this->getAdviceFromResponseBody($response->body);
+    }
+
+    /**
+     * Return the best advice for the other player
+     * Making it the worst advice for the current player
+     *
+     * @param  Game $game
+     *
+     * @return array
+     */
+    public function getWorstAdvice(Game $game): array
+    {
+        $body = [
+            'Grid' => $game->getBoard(),
+            'Player' => $this->gameManager->getEnnemyPlayer($game->getCurrentPlayer()),
```

Comment:
Can the worst advice really be the opponent's best advice? I don't understand the logic...

Reply:
My worst move is the move that make the opponent win.

Reply:
Seen together